openvpn_api/events/__init__.py

```diff
@@ -0,0 +1,17 @@
+import importlib
+import typing
+
+from openvpn_api.events import client
+
+event_types = [importlib.import_module(".client", __name__)]
```

Comment:
Well, I was more of a module guy than a class guy, but it's changed nowadays. ( Long time from May :D )
I don't exactly remember why I chose module, but I guess it was because of the singleton nature of the event module/class, which would make more sense for it to be a module.